packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -15,5 +16,8 @@ export const template = html<TableHeader>`
         ${when(x => x.sortDirection === TableColumnSortDirection.descending, html`
             <${iconArrowDownTag} class="sort-indicator" aria-hidden="true"></${iconArrowDownTag}>
         `)}
+        ${when(x => x.isGrouped, html`
+            <${iconTwoSquaresInBracketsTag} class="grouped-indicator"></${iconTwoSquaresInBracketsTag}>
```

Comment:
If you resize the table width to the point of column names being truncated, it looks like the grouped indicator icon is shrinking from its default size.

**Original**

![image](https://user-images.githubusercontent.com/10500124/232910043-d6db24f1-a288-426c-bfdf-c4c37d7b5fdf.png)


**Shrunk**

![image](https://user-images.githubusercontent.com/10500124/232910249-6dc11fd2-d7ac-4fe4-9a09-0406f83eeeb2.png)

Reply:
I happened to notice the same issue with the sort icon, which is already in main. Setting `flex-shrink: 0` on those icons should fix the problem.

Reply:
Had to set some flex settings on the icon.